src/Symfony/Component/HttpKernel/Tests/Controller/ArgumentResolver/RequestPayloadValueResolverTest.php

```diff
@@ -184,9 +187,9 @@ public function testQueryNullPayloadAndNotDefaultOrNullableArgument()
         $validator->expects($this->never())
             ->method('validate');
 
-        $resolver = new RequestPayloadValueResolver(new Serializer(), $validator);
+        $resolver = new RequestPayloadValueResolver(new Serializer([new ObjectNormalizer()]), $validator);
 
-        $argument = new ArgumentMetadata('valid', RequestPayload::class, false, false, null, false, [
+        $argument = new ArgumentMetadata('valid', QueryPayload::class, false, false, null, false, [
```

Comment:
Same here, you should add new tests instead of modifying the current ones that are testing the RequestPayload

Reply:
Again, same thing here. There are tests for both `Request` and `Query` cases. All of them uses `RequestPayload`. This is irrelevant because `MapQueryString` or `MapRequestPayload` is used properly according to test case. My intention was only to fix readability.

Reply:
I see 👍